src/SceneGraphDebugCommands.ts

```diff
@@ -54,7 +54,7 @@ export class SceneGraphDebugCommands {
             await this.logCommandOutput(async (commandController) => commandController.free());
         }));
 
-        subscriptions.push(vscode.commands.registerCommand('extension.brightscript.genKey', async () => {
+        subscriptions.push(vscode.commands.registerCommand('extension.brightscript.genkey', async () => {
```

Comment:
@chrisdp where's the corresponding command in package.json? I only see the entry for `showkey`.

Reply:
https://github.com/rokucommunity/vscode-brightscript-language/pull/313/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R1516